matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/auth/AuthenticationService.kt

```diff
@@ -53,15 +53,19 @@ interface AuthenticationService {
      * Return a LoginWizard, to login to the homeserver. The login flow has to be retrieved first.
      *
      * See [LoginWizard] for more details
+     * @param enableRefreshTokenAuth whether to enable refresh token based auth (in beta, hence default is false for now)
+     * instead of long lived access tokens.
      */
-    fun getLoginWizard(): LoginWizard
+    fun getLoginWizard(enableRefreshTokenAuth: Boolean = false): LoginWizard
```

Comment:
Should this flag enabled on MatrixConfiguration instead? It would avoid passing this everywhere and directly be injected in the few places where it makes sense.

Reply:
I actually had it that way initially. @bmarty felt it would be better to be a more explicit part of the api on the `AuthenticationService` vs a config people might not know about. I don't really have a strong opinion on it. :)